### PR TITLE
Feature SliceFieldPrint 2D

### PR DESCRIPTION
**Pull request by Heikman**
2D support for SliceFieldPrinter plugin.

### CODE REVIEW

**Comment by PrometheusPi** [comment]
waiting for #844 to be merged

File: src/picongpu/include/plugins/SliceFieldPrinterMulti.hpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013-2014 Heiko Burau, Rene Widera, Felix Schmitt,
+ * Copyright 2013-2015 Heiko Burau, Rene Widera, Felix Schmitt,
```
Comment (PrometheusPi) [inline src/picongpu/include/plugins/SliceFieldPrinterMulti.hpp]: What did you change in this file to justify a update of the license year? :-)

**Comment by PrometheusPi** [comment]
@Heikman #844 is merged. Could you please rebase against `dev`?

**Comment by Heikman** [comment]
done

**Comment by PrometheusPi** [comment]
@Heikman 
 - Please remove ` src/picongpu/include/plugins/SliceFieldPrinterMulti.hpp
`from the list. You changed nothing in this file exept the license year. 
 - Did you want to refactor `--<field>_slice.slicePoint` to take cell numbers instead of a float between 0 and 1? (Could be included in a fllow up pull request).

**Comment by PrometheusPi** [comment]
The 2D support works great:

2dslices

File: src/picongpu/include/plugins/SliceFieldPrinterMulti.hpp [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013-2014 Heiko Burau, Rene Widera, Felix Schmitt,
```
Comment (ax3l) [inline src/picongpu/include/plugins/SliceFieldPrinterMulti.hpp]: *removing* something from the copyright line is not correct. pls revert

File: src/picongpu/include/plugins/SliceFieldPrinterMulti.tpp [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013-2014 Heiko Burau, Rene Widera, Felix Schmitt,
+ * Copyright 2013 Heiko Burau, Rene Widera, Felix Schmitt,
```
Comment (ax3l) [inline src/picongpu/include/plugins/SliceFieldPrinterMulti.tpp]: removing something from the copyright line is not correct. pls update to *2013-2015* in this file

**Comment by ax3l** [comment]
pretty sweet.

@PrometheusPi did you try the *multi* feature, too? did you also try 2D support (creating an array from the 2D plane)?

**Comment by ax3l** [comment]
careful, if `--<field>_slice.slicePoint` is a cell number it won't be very useful for co-moving window setups I guess.

alternatively, introduce a `--<field>_slice.comoving` flag that increases the `y` selection with each timestep accordingly.

**Comment by PrometheusPi** [comment]
@ax3l Yes, i tried the multi feature as well.

File: src/picongpu/include/plugins/SliceFieldPrinterMulti.tpp (hunk 3 above)
Comment (PrometheusPi) [inline src/picongpu/include/plugins/SliceFieldPrinterMulti.tpp]: yes add `-2014` only `-2015` was wrong.

Comment (ax3l) [inline src/picongpu/include/plugins/SliceFieldPrinterMulti.tpp]: @PrometheusPi that comment is a bit confusing.
he updated the file. so the line `2013-2014` that he falsely removed should read `2013-2015`.

**Comment by Heikman** [comment]
just fixed the copyright years. thanks for advice

**Comment by Heikman** [comment]
I opened an issue for the missing features you mentioned.

**Comment by ax3l** [comment]
well done, thx!

**Comment by PrometheusPi** [comment]
great job - I will merge

**Pull request merged**